src/components/subscriptions/expiration/SubscriptionExpirationBanner.jsx

```diff
@@ -45,17 +79,35 @@ const SubscriptionExpirationBanner = () => {
     alertType = 'danger';
   }
 
+  let actions = [
+    <Button onClick={() => { window.open(configuration.ENTERPRISE_SUPPORT_URL, '_blank'); }}>
+      Contact customer support
+    </Button>,
+  ];
+  if (props.subscriptionDetails && daysUntilExpiration <= SUBSCRIPTION_DAYS_REMAINING_SEVERE) {
+    actions = undefined;
+  }
```

Comment:
nit: change to `const actions = []` and use `actions.push` for when the `Button` should be included. saves the overriding of the `actions` variable to `undefined`.